src/tests/unit/tests/background/tab-controller.test.ts

```diff
@@ -41,20 +37,21 @@ describe('TabControllerTest', () => {
             .verifiable(Times.once());
     }
 
-    function createTabControllerWithoutFeatureFlag(tabcContextMap: TabToContextMap): TabController {
+    function createTabControllerWithoutFeatureFlag(tabContextMap: TabToContextMap): TabController {
         return new TabController(
-            tabcContextMap,
+            tabContextMap,
             mockBroadcasterStrictMock.object,
             mockChromeAdapter.object,
             mockDetailsViewController.object,
             tabContextFactoryMock.object,
-            LoggerStub,
+            loggerMock.object,
         );
     }
 
     beforeEach(() => {
-        logMock = Mock.ofInstance((msg: string) => {});
-        LoggerStub.log = logMock.object;
+        loggerMock = Mock.ofType<Logger>();
+        // logMock = Mock.ofInstance((msg: string) => {});
+        // LoggerStub.log = logMock.object;
```

Comment:
Forgot to remove the commented code here?

Reply:
thanks for catching that.